e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/util/client/RepositoryClient.java

```diff
@@ -140,4 +172,23 @@ public void deleteNode(String nodeId)
                 .contentType("application/json")
                 .when().delete(uri);
     }
+
+    public String classifyNode(String nodeId, String securityGroupId, String securityMarkId)
```

Comment:
I would call this `secureNode` since `classify` has a very specific meaning in AGS (using the classification group).

Reply:
changed